### PR TITLE
Bump to node 10

### PR DESCRIPTION
**Pull request by jason-fox**
Node 8 leaves LTS at the end of the year. The default node version should be switched to Node Dubnium for the upcoming December FIWARE 7.8.1 Patch release.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -19,7 +19,7 @@
 # For those usages not covered by the GNU Affero General Public License
 # please contact with: [daniel.moranjimenez@telefonica.com]
 
-ARG NODE_VERSION=8.16.1-slim
+ARG NODE_VERSION=10.17.0-slim
```
Comment (fgalan) [inline docker/Dockerfile]: As other times (see https://github.com/telefonicaid/iotagent-ul/releases/tag/1.11.0) an entry in CHANGES_NEXT_RELEASE should be included about the changes, such as:

```
Upgrade NodeJS version from X.Y.Z to W.U.V in Dockerfile
```

Comment (jason-fox) [reply docker/Dockerfile]: Fixed d44a5df

**Comment by fgalan** [comment]
It seems there is a conflict in CHANGES_NEXT_RELEASE... as soon as it gets solved, I'll merge.

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -0,0 +1,3 @@
+- Use AMQP durable option in assertExchange
+- Use device apikey if exists in getEffectiveApiKey for command handling
+- Upgrade NodeJS version from 8.16.1 to 10.17.0 in Dockerfile due to Node 8 End-of-Life
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: I think the conflict with master wasn't correctly solved... Note that CHANGES_NEXT_RELEASE in mater  (https://github.com/telefonicaid/iotagent-ul/blob/master/CHANGES_NEXT_RELEASE) is empty, so this PR should only include L3.

L1-L2 where included in last release (https://github.com/telefonicaid/iotagent-ul/releases/tag/1.12.0)

Comment (jason-fox) [reply CHANGES_NEXT_RELEASE]: Fixed 33cbd86.

**Pull request merged**